index.html

```diff
@@ -29,20 +29,33 @@
   <div class="row featurette text-center">
     <h2 class="text-center">Babel transforms your JavaScript</h2>
 
+    <h3 class="lead text-center">
+      {% highlight javascript %}
+      npm install --save-dev babel-cli babel-preset-es2015
+      {% endhighlight %}
+      {% highlight javascript %}
+      ./node_modules/.bin/babel index.js --presets=es2015
```

Comment:
I have no idea haha.. What looks clearer?

If we wanted conciseness I would of done `npm i -D babel-{cli, preset-es2015}`